src/server/IoDispatchers.cpp

```diff
@@ -296,16 +297,50 @@ PCONSOLE_API_MSG IoDispatchers::ConsoleHandleConnectionRequest(_In_ PCONSOLE_API
             HANDLE serverHandle;
             THROW_IF_FAILED(Globals.pDeviceComm->GetServerHandle(&serverHandle));
 
+            wil::unique_hfile signalPipeTheirSide;
+            wil::unique_hfile signalPipeOurSide;
+
+            THROW_IF_WIN32_BOOL_FALSE(CreatePipe(signalPipeOurSide.addressof(), signalPipeTheirSide.addressof(), nullptr, 0));
+
+            // Give a copy of our own process handle to be tracked.
+            wil::unique_process_handle ourProcess;
+            THROW_IF_WIN32_BOOL_FALSE(DuplicateHandle(GetCurrentProcess(),
+                                                      GetCurrentProcess(),
+                                                      GetCurrentProcess(),
+                                                      &ourProcess,

```

Comment:
For the sake of consistency this should probably be `ourProcess.addressof()`.
The `operator&` is an alias for `.put()` and performs a `.reset()` (deallocation) and returns the same as `.addressof()`. And right above this you use `.addressof()` already.

Personally I don't understand the existence of `.addressof()` in wil - it seems like a macro-optimization to me.

Reply:
It's probably a COM reason. It just feels like one. I dunno. Made it line up.